SecurityPkg/VariableAuthenticated/SecureBootConfigDxe/SecureBootConfigImpl.c

```diff
@@ -2652,8 +2661,53 @@ UpdateDeletePage (
       //
       // Display GUID and help
       //
-      GuidToString (&Cert->SignatureOwner, GuidStr, 100);
-      GuidID = HiiSetString (PrivateData->HiiHandle, 0, GuidStr, NULL);
+
+
+      // Only half of the buffer size may be used for the names
+      // Because it needs to be resized to 16b which
+      // will cause it to grow in size 2x
+
+      CertificateInfoReadSuccess = X509GetSubjectName(
+        (UINT8*)Cert->SignatureData,
+        (UINTN)CertList->SignatureSize,
+        (UINT8*)CertificateInfoStr,
+        &CertificateInfoStrSizeHalf
+      )
+
+      if(!CertificateInfoReadSuccess)
+      {

```

Comment:
```suggestion
      if (!CertificateInfoReadSuccess)   {
```

Please conform to the project's coding style.